Impossible to download with recalbox 4.1

```
WARNING: no hash file for recalboxOS-4.0.0.zip
--2017-04-14 11:15:12--  http://archive.recalbox.com/4.1/rpi3/unstable/last/boot.tar.xz
Résolution de archive.recalbox.com (archive.recalbox.com)… 163.172.147.40, 51.15.141.52
Connexion à archive.recalbox.com (archive.recalbox.com)|163.172.147.40|:80… connecté.
requête HTTP transmise, en attente de la réponse… 404 Not Found
2017-04-14 11:15:12 erreur 404 : Not Found.

--2017-04-14 11:15:12--  http://sources.buildroot.net/boot.tar.xz
Résolution de sources.buildroot.net (sources.buildroot.net)… 176.9.16.109
Connexion à sources.buildroot.net (sources.buildroot.net)|176.9.16.109|:80… connecté.
requête HTTP transmise, en attente de la réponse… 404 Not Found
2017-04-14 11:15:12 erreur 404 : Not Found.

package/pkg-generic.mk:113 : la recette pour la cible « /media/musique/Roms/recal_rasp/recalbox-rasplex/output/build/recalboxOS-4.1-unstable/.stamp_downloaded » a échouée
make[1]: *** [/media/musique/Roms/recal_rasp/recalbox-rasplex/output/build/recalboxOS-4.1-unstable/.stamp_downloaded] Erreur 1
Makefile:36 : la recette pour la cible « _all » a échouée
make: *** [_all] Erreur 2
```

--- a/package/recalboxOS/recalboxOS.mk
+++ b/package/recalboxOS/recalboxOS.mk
@@ -15,8 +15,8 @@ ifeq ($(BR2_PACKAGE_RECALBOXOS_UNSTABLE),y)
 else ifeq ($(BR2_PACKAGE_RECALBOXOS_NEXT),y)
 	RECALBOXOS_VERSION = 4.1-unstable
 	RECALBOXOS_EXTRA_DOWNLOADS = \
-		http://archive.recalbox.com/4.1/$(BR2_ARCH)/unstable/last/boot.tar.xz \
-		http://archive.recalbox.com/4.1/$(BR2_ARCH)/unstable/last/root.tar.xz
+		http://archive.recalbox.com/updates/v1.0/unstable/$(BR2_ARCH)/boot.tar.xz \
+		http://archive.recalbox.com/updates/v1.0/unstable/$(BR2_ARCH)/root.tar.xz
 else
 	RECALBOXOS_VERSION = $(RECALBOXOS_RELEASE)
 endif

